sktime/classification/kernel_based/_rocket_classifier.py

```diff
@@ -109,7 +104,7 @@ def fit(self, X, y):
         self._is_fitted = True
```

Comment:
this should also go, it's in the base class too

Reply:
ac47ae9c